macros/recursive_dag.sql

```diff
@@ -31,6 +33,8 @@ all_relationships (
     child_resource_type,
     child_model_type,
     child_materialized,
+    child_access,
+    child_is_public,
```

Comment:
Why the need for both `access` and `is_public` - isn't `is_public` just `if access == 'public'`?

Reply:
I think just to allow for future rules that use `access`! can remove for now if we don't think it's valuable